fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/api/ChargesApiResourceSwagger.java

```diff
@@ -52,7 +52,7 @@ static final class GetChargesTimeTypeResponse {
             private GetChargesTimeTypeResponse() {}
 
             @Schema(example = "1")
-            public Long id;
+            public Integer id;
```

Comment:
This looks wrong to me...It should be Long based on the EnumOptionData line:36

Reply:
pls see ChargeTimeType.java  line:42 too

Reply:
If you want to use Integer, you need to change the EnumOptionData class and the ChargeEnumerations class chargeTimeType method which is converting the getvalue from int to Long and creates the EnumOptionData.

Either you are changing everywhere or you are changing back from Integer to Long in the swagger

Ultimately we cannot have different value in the swagger and in the DTO which fills with values.

Reply:
Done.